example/index.html

```diff
@@ -464,7 +464,7 @@ <h2>Copyright and License Element</h2>
     draft-bhutton-json-schema-00</cite>, Wright, A., H. Andrews; B. Hutton, G.
     Dennis, "JSON Schema: A Media Type for Describing JSON Documents", Work in
     Progress, December 8, 2020,
-    https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00
+    <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00</a>
```

Comment:
?

Reply:
Not sure what the question is here, added the href to bib refs as well.